src/routes/auth.js

```diff
@@ -102,4 +103,28 @@ router.post(
 	})
 )
 
+router.post(
+	'/token',
+	isAuthenticated(),
+	isAdmin,
```

Comment:
not only restricted to admins, we'll be refreshing the token for all kinds of users

Reply:
yes please, thank you